source/overview/product.rst

```diff
@@ -2,70 +2,154 @@
+Mattermost keeps vital communications like access to mobile and desktop apps behind your firewall and can be deployed using `dozens of security features <https://docs.mattermost.com/overview/security.html>`_ vetted by top security researchers. 
+Data stays on servers you control and is encrypted using keys you control.
 
-- **Scalability** - Grow from dozens of users to tens of thousands on the same server. Built on a high performance, single-tenant infrastructure, Mattermost E20 offers cluster-based high availability deployments with horizontal scaling and advanced performance monitoring.
+Configurability 
+~~~~~~~~~~~~~~-
 
-About the Mattermost open source project
+Mattermost allows organizations to adapt deployments to their needs, 
+preferences, 
+and existing systems. 
+The open source Mattermost server enables advanced white labelling and customization with complete access to RESTful server APIs, 
+drivers, 
+webhooks, 
+and hundreds of third party extensions.
+
+Scalability 
+~~~~~~~~~~~~
+
+Mattermost can scale from dozens to tens of thousands of users on the same server. 
+The Mattermost E20 offers cluster-based high availability deployments with horizontal scaling and advanced performance monitoring atop single-tenant infrastructure with high performance.
+
+
+About the Mattermost Open Source Project
 ----------------------------------------------
 
-At its core, Mattermost is an open source, private cloud alternative to proprietary SaaS messaging for teams. The software, developed in partnership with over 500 contributors from around the world, rivals the productivity of SaaS solutions. We achieve this while offering organizations superior control, data sovereignty, configurability, freedom from lock-in, and enhanced security by keeping vital communications behind your firewall.
+Mattermost is an open source, 
+private cloud alternative to proprietary SaaS messaging for teams. 
+The software has been developed in partnership with over 500 contributors from around the world and easily rivals the productivity of SaaS solutions. This productivity is achieved with superior control, 
+data sovereignty, 
+configurability, 
+freedom from lock-in, 
+and enhanced security that keeps vital communication behind your firewall.
+
+Roadmap
+~~~~~~~
 
-Core committers, including both community contributors and paid staff at Mattermost, Inc., determine the project roadmap. For enterprises with needs beyond the scope of the open source project, commercial "Enterprise Edition" extensions are available from Mattermost, Inc. Partnership with our core committer community, along with revenue from Enterprise Edition, ensures the continued improvement of all editions.
+The Mattermost roadmap is drawn by community contributors and paid staff at Mattermost, Inc.
```

Comment:
maybe we should change this to "influenced by community contributors and users of Mattermost", since technically the roadmap is drawn by us but influenced by others

Reply:
done